third_party/python.bzl

```diff
@@ -71,3 +71,14 @@ def tensorboard_python_workspace():
       strip_prefix = "six-1.10.0",
       build_file = str(Label("//third_party:six.BUILD")),
   )
+
+  native.new_http_archive(
+      name = "org_pythonhosted_portpicker",
```

Comment:
One last small request. Would it be possible to name this org_python_pypi_portpicker since portpicker has no affiliation with pythonhosted.org?